pytest.ini

```diff
@@ -14,4 +15,4 @@ DJANGO_CONFIGURATION = Test
 
 
 [pytest-watch]
-ignore = frontend .git docs test-with-nginx
+ignore = frontend .git docs test-with-nginx .cache htmlcov
```

Comment:
These changes are actually irrelevant. I just noticed it whilst working on this. Sorry.